src/lucky_migrator/references_helper.cr

```diff
@@ -4,8 +4,17 @@ module LuckyMigrator::ReferencesHelper
       ""
     elsif on_delete == :do_nothing
       " REFERENCES #{table_name}"
-    elsif CreateForeignKeyStatement::ALLOWED_ON_DELETE_STRATEGIES.includes?(on_delete)
-      " REFERENCES #{table_name}" + " ON DELETE " + "#{on_delete}".upcase
+    else
+      " REFERENCES #{table_name}" + " ON DELETE " + on_delete_sql(on_delete)
+    end
+  end
+
+  private def on_delete_sql(on_delete : Symbol?) : String
+    case on_delete
+    when :nullify
+      "SET NULL"
```

Comment:
Nice! Would this change have to be applied here as well https://github.com/luckyframework/lucky_migrator/blob/master/src/lucky_migrator/create_foreign_key_statement.cr?

Reply:
Yes it should be done there too. Nice catch :)

Reply:
Thanks for pointing this out! I added a commit that makes the `CreateForeignKeyStatement` class use the method from `ReferencesHelper`.

I guess there is some potential to move more shared code into that helper, but I think this is out of scope for this PR.